tests/suites/test_suite_rsa.function

```diff
@@ -158,6 +170,25 @@ void rsa_init_free(int reinit)
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
```

Comment:
I'm not sure if we really need this test. We just added two simple getters and the negative cases have been covered by `rsa_invalid_param`.

Reply:
Initially I planned to add a test case which validates the value of `padding mode` and `hash_id`. But I didn't find proper area to check those two values in `test_suite_rsa.function`. That's why I added this test function. Now I realize we only have to call this function once to test its functionality. This should be enough. I'll remove redundant tests.